### PR TITLE
Main page dark mode

### PR DESCRIPTION
**Pull request by MRodriguez-M**
Implemented feature so that the main page has a toggle option for dark mode. Addresses issue #10

### CODE REVIEW

File: main.css [hunk 1]
```diff
@@ -91,8 +92,11 @@ body{
     overflow: hidden;
 }
 
+:root {
```
Comment (sarmadhamdani02) [inline main.css:95]: I believe placing the variables at the beginning of the CSS file would be a better approach. Additionally, using more descriptive names for the variables would enhance the code's readability, especially for new contributors, including beginners. And if I am not getting this wrong this `background-color` is for the toggle button's background color 🤔 if so, naming it as `toggleBtn-background-color` will be a better approach.

File: script.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    }
+    for(let i = 0; i < gameIcon.length; i++) {
+      gameIcon[i].style.setProperty("--background-color", "#22303C");
+    }
+
+    navBar[0].style.backgroundColor = "#03DAC5";
+    navBar[0].style.color = "#121212";
+    for(let i = 0; i < navItem.length; i++) {
+        navItem[i].style.color = "#121212";
+    }
```
Comment (sarmadhamdani02) [inline script.js:22]: I think the color `#121212` is too dark. Adding a bit less dark color like `#212628` will look much more appealing. Try playing with a bit less dark colors and change it if you feel like it.

Comment (sarmadhamdani02) [reply script.js:22]: Apart from this, I think everything is amazing 👏👏

**Review by sarmadhamdani02 — CHANGES_REQUESTED** [review]

**Comment by sarmadhamdani02** [comment]
Thank you so much @MRodriguez-M  for the great work 👏👏

**Comment by MRodriguez-M** [comment]
> Hey @MRodriguez-M 👋
> 
> I've recently opened a new issue regarding the dark mode feature. It appears that after implementing the dark mode, the responsiveness of the Navbar has been affected. As no one knows the dark mode code better than you, I thought you might be the best person to take a look at issue #41, if you're available.
> 
> I know that you are already working on other issues so if you don't want to do that. There is absolutely no problem. 😊
> 
> Regards.

Yeah, I can take care of that. When building the dark mode feature, I didn't fully take responsiveness into account so now's a great time to do it. I'll go leave a message on the issue so it can be assigned to me

**Pull request merged**